dockerfiles/centos_7

```diff
@@ -1,12 +1,10 @@
-ARG BASE_IMAGE
-FROM ${BASE_IMAGE}centos:7
+FROM centos:7
 LABEL org.opencontainers.image.authors="artembo@me.com; piligrim@rootnix.net"
 
 ARG TNT_VER \
-    GC64
```

Comment:
What's the poing of replacing GC64 with LUAJIT_ENABLE_GC64 and then using LUAJIT_ENABLE_GC64 to set GC64?

Reply:
Variable LUAJIT_ENABLE_GC64 set before this modification and functionality GC64 variable duplicates it.